sql/tokenize.cc

```diff
@@ -38,28 +36,52 @@ TokenizeOptionsDescription()
     addAuto("value", &TokenizeOptions::value,
             "`value` (if not set to `null`) will be used instead of "
             "token counts for the values of the columns in the output row.");
-    addAuto("min_token_length", &TokenizeOptions::min_token_length,
+    addAuto("minTokenLength", &TokenizeOptions::minTokenLength,
             "Minimum number of characters in a token for it to be output or "
             "included as part of an ngram");
-    addAuto("ngram_range", &TokenizeOptions::ngram_range,
+    addAuto("ngramRange", &TokenizeOptions::ngramRange,
             "Specifies the complexity of n-grams to return, with the "
             "first element corresponding to minimum length and the "
             "second to maximum length.  "
             "`[1, 1]` will return only unigrams, while `[2, 3]` will "
             "return bigrams and trigrams, where tokens are joined by "
             "underscores. For example, "
-            "`tokenize('Good day world', {splitchars:' ', ngram_range:[2,3]})`"
+            "`tokenize('Good day world', {splitChars:' ', ngramRange:[2,3]})`"
             "will return the row `{'Good_day': 1, 'Good_day_world': 1, 'day_world': 1}`");
+
+    onUnknownField = [] (TokenizeOptions * options,
+                         JsonParsingContext & context)
+    {
+        if(context.fieldName() == "min_token_length") {
```

Comment:
Aren't we better to warn the user and then accept the argument under the old name?